commands/new.go

```diff
@@ -60,8 +61,16 @@ Ensure you run this within the root directory of your site.`,
 					return create.NewContent(h, contentType, args[0], force)
 				},
 				withc: func(cmd *cobra.Command, r *rootCommand) {
+					cmd.ValidArgsFunction = func(cmd *cobra.Command, args []string, toComplete string) ([]string, cobra.ShellCompDirective) {
+						if len(args) != 0 {
+							return []string{}, cobra.ShellCompDirectiveNoFileComp
+						}
+						return []string{}, cobra.ShellCompDirectiveNoFileComp | cobra.ShellCompDirectiveFilterDirs
+					}
 					cmd.Flags().StringVarP(&contentType, "kind", "k", "", "content type to create")
+					_ = cmd.RegisterFlagCompletionFunc("kind", cobra.FixedCompletions(kinds.AllKinds, cobra.ShellCompDirectiveNoFileComp))
```

Comment:
I have not looked at all usage of AllKinds in here, but I would say that for completion, I think the current best approach would be to:

* Use AllKindsInPages in the above case and similar.
* Use AllKinds for the `disableKinds`.

Reply:
Cheers, changed this one to `AllKindsInPages`. `AllKinds` was already in use in `disableKinds`.